README.md

```diff
@@ -63,11 +65,23 @@ You can also configure an instance of a client directly.
 
 ```ruby
 client = IEX::Api::Client.new(
-  publishable_token: 'token',
+  publishable_token: 'publishable_token',
+  secret_token: 'secret_token',
   endpoint: 'https://sandbox.iexapis.com/v1'
 )
 ```
 
+### Make general requests
+
```

Comment:
It's unclear what's not implemented.

"Public endpoints that aren't yet supported by the client can be called using `client.get`, `post`, `put` and `delete` methods ... Private endpoints that aren't yet supported by the client can be called using ..."